demo/web/src/main/notebooks/B. Data science, ML, & AI/B1 Pandas and Deephaven.md

```diff
@@ -0,0 +1,394 @@
+# **Dynamic data and Pandas**
+
+This notebook will demonstrate how to switch from DataFrames to tables in Deephaven.
```

Comment:
This notebook familiarizes you with some of the options for working with Pandas; in particular,  it demonstrates how to switch from DataFrames to tables in Deephaven.